### ISSUE TITLE
Docker images issues on Debian due to CentOS 6

### ISSUE THREAD

**Issue by edalex-ian** [kaltura/platform-install-packages#702]
- [x] I've read the [guidelines for Contributing to Kaltura Projects](https://github.com/kaltura/platform-install-packages/blob/master/CONTRIBUTING.md)
- [x] I've read the [installation guides](https://github.com/kaltura/platform-install-packages#documentation-and-guides) and [troubleshooting FAQ](https://github.com/kaltura/platform-install-packages/blob/master/doc/kaltura-packages-faq.md)
- [x] This is not a support request or question that should be posted on the [Kaltura forum](https://forum.kaltura.org) 

## Bug report

Your docker image fails to start on Debian based setups due to known issues running CentOS 6 docker images on debian. Is it possible someone could rework it to CentOS 7 (or ideally the current 8)?

I tried, by my CentOS knowledge was not sufficient - or more so, seeing I'm trying to use it to come up to speed with Kaltura, I think my knowledge there is more so insufficient.

See also #646

**Comment by ChrisB9** [comment]
Did you get it working under CentOS 7 or 8 in docker?

**Comment by edalex-ian** [comment]
Nope, afraid not. Had to resort to using the SaaS platform. :disappointed:
